### ISSUE TITLE
TypeError: null is not an object (near '...}))}(t,"$.divider")).style.marginRight=

### ISSUE THREAD

**Issue by Mariusthvdb** [thomasloven/lovelace-fold-entity-row#161]
My Home Assistant version: 2021.6.0b5

Fold-entity-row version (FROM BROWSER CONSOLE):
20.0.4
What I am doing:
have 4 entities cards using fold-entity-row to fold some other entities

What I expected to happen:
no error
What happened instead:

see screenshot. Folds do work as expected
this is in Safari/Mac. Error isn't always there, and only shows after the view has been loaded for some time. Havent seen it in Chrome yet.
**Minimal** steps to reproduce:

```yaml
# The least ammount of code possible to reproduce my error
  - type: entities
    title: Summary system automation
    show_header_toggle: false
    entities:
      - some entity
      - some other entity
      - type: custom:fold-entity-row
        head:
          type: section
          label: Sense automations
        padding: 0
        entities:
          - entity: automation.sense_family_home_change
            secondary_info: last-triggered
# End of code
```

Error messages from the browser console:
```
Unhandled Promise Rejection: TypeError: null is not an object (near '...}))}(t,"$.divider")).style.marginRight="...')
```
---

<img width="736" alt="Schermafbeelding 2021-06-02 om 12 36 42" src="https://user-images.githubusercontent.com/33354141/120466371-45036980-c39f-11eb-9b0f-9f46af87ba0c.png">

**By putting an X in the boxes ([X]) below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [X] Have made sure I am using the latest version of the plugin.
- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.
- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by thomasloven** [comment]
I think this is fixed in 20.0.5?

**Comment by Mariusthvdb** [comment]
havent seen it, so closing, and if it returns Ill let you know ;-)
thanks.
